vendors/alastria/alastriaIdCreations/custom-alastriaIdCreations/example-of-alastriaIdCreations.json

```diff
@@ -1,3 +1,4 @@
+    "alastriaAICComplete":"eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NksiLCJraWQiOiJkaWQ6YWxhOnF1b3I6cmVkdDoweDEyZWVhQ0NBOWVFYkI3OGVCOTdkN2NhYzZiI2tleXMtMSIsImp3ayI6IjB4MGNlNWVkYzIwNGFlOTc3ZDZkNjlmY2M2YjMzOWE1N2I1NDRlZjRkMDk0OTI5ZjNlYjkxOGE3N2RmYmEyNjk5OGU3N2VjNWIyOWEzNzlkZDU5YmJlOTYyZTJiMjI3NDUxNDNhYTJiNTJjNGE4NzViNzQ5MmMxMzdlZWViZjQwNTUifQ.eyJAY29udGV4dCI6WyJodHRwczovL2FsYXN0cmlhLmdpdGh1Yi5pby9pZGVudGl0eS9hcnRpZmFjdHMvdjEiLCJodHRwczovL3czaWQub3JnL2RpZC92MSIsIkpXVCJdLCJ0eXBlIjpbIkFsYXN0cmlhSWRlbnRpdHlDcmVhdGlvbiIsInR5cGUxIiwidHlwZTIiXSwiY3JlYXRlQWxhc3RyaWFUWCI6IjB4ZjkwMTg4ODIwM2YxODA4MzA5MjdjMDk0YmQ0YTJjODRlZGI5N2JlNWJlZmY3Y2QzNDFiZDYzNTY3ZTczZjhjOTgwYjkwMTI0NmQ2OWQ5OWEwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDIwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDBjNDUwMzgyYzFhMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAyMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwODAzMjY1MzUzMDM3NjE2NjMwMzEzMTM2Mzc2MzM5Mzg2MTM2NjE2MzYzNjMzMDYzNjQzNDM2NjE2MjMyNjEzMjM1MzY2NDY0MzY2MjM2NjMzNjM5NjU2MzMxNjMzMDYzMzIzODY2MzgzMDY2NjIzNjMyNjUzMTY2Mzc2NDM3MzAzMjMzMzMzNzM2Mzg2MjMwNjMzNTM4NjQ2MjYyNjQ2MTYzMzE2NjYzMzMzNTM4NjIzODMxMzQzMTYzMzAzNzM1NjEzNTMyMzAzNDM4MzM2MzY2MzkzNzM3Mzk2NTM0NjU2MTM5Mzg2NDMxMzM2NDY2MzIzODMzMzM2NjMzMzczNjM3MDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAxY2EwZTNlMDVhYmIxNTkyYmQxNWRjNzJlNTdkODkzMGMyNGVmZGQwZTI4NjQzN2EzNGNhMmYxMTgzNzJhZDQyOGIyNWEwNDgwYWM4MTZjOTNhNWMxZThlZmYxZGZkOTZmODUwZGM2NjRjNWRhYTFmZjk0YTg4YmRmNzg5YjA2NTJlYmZkMyIsImFsYXN0cmlhVG9rZW4iOiJleUowZVhBaU9pSktWMVFpTENKaGJHY2lPaUpGVXpJMU5rc2lmUS5leUpwYzNNaU9pSmthV1E2WVd4aE9uRjFiM0k2Y21Wa1ZEb3dlR0l3TkRZeU5qQXhZVEUzTlRneFpXUXhZalZrWkdFNE5qSTNNbUZsWlRWbE5EbGxNbVkxWmpjaUxDSm5kM1VpT2lKb2RIUndjem92TDNKbFozVnNZWEl1ZEdWc2MybDFjeTVpYkc5amEyTm9ZV2x1WW5sbGRtVnlhWE11YVc4Nk1qQXdNQ0lzSW1OaWRTSTZJbWgwZEhCek9pOHZjMlZ5ZG1salpYQnliM1pwWkdWeUxtRnNZWE4wY21saExtSnNiMk5yWTJoaGFXNWllV1YyWlhKcGN5NXBieTloY0drdmJHOW5hVzR2SWl3aWFXRjBJam94TlRreE56azNOVE0zTENKaGJta2lPaUp5WldSVUlpd2libUptSWpveE5UWXpOemd5TnpreUxDSmxlSEFpT2pFMU5qTTNPRE16T1RJc0ltcDBhU0k2SW5wbE1qazRlVFF5YzJKaEluMC53Z2UxWi1TNmdReVNfTHVWNTBJUW55S05OaDF0MUhONF8ycWF0aTFHZWJ6Y1hvYXBSbG9zMjNSYlJqSk9qOWF0dEE3SVJUX0J6dHdYT0hNRWE2YThNdyIsInB1YmxpY0tleSI6IjB4MGNlNWVkYzIwNGFlOTc3ZDZkNjlmY2M2YjMzOWE1N2I1NDRlZjRkMDk0OTI5ZjNlYjkxOGE3N2RmYmEyNjk5OGU3N2VjNWIyOWEzNzlkZDU5YmJlOTYyZTJiMjI3NDUxNDNhYTJiNTJjNGE4NzViNzQ5MmMxMzdlZWViZjQwNTUiLCJqdGkiOiJodHRwczovL3d3dy5lbXByZXNhLmNvbS9hbGFzdHJpYS9jcmVkZW50aWFscy8zNzM0IiwiaWF0IjoxNTYzNzgyNzkyLCJleHAiOjE1NjM3ODMzOTIsIm5iZiI6MTIzNDU2Nzg5fQ.5fNE42w2F6KkzzbpqGiLIjxV8ztH-CPFaNQ097np03V1r-Z7Wz9fg70BN3bwWRThoVdd1SJ-5UgTsWoAtEM1Bg",
```

Comment:
Hola! Simplemente una indicación, la última parte del did, que se refiere a la address del AlastriaProxy, no debe comenzar con "0x" si queremos que funcionen correctamente las librería tal y como se especifica en la [wiki ](https://github.com/alastria/alastria-identity/wiki/Alastria-DID-Method-Specification-(Quorum-version)#21-alastria-did-scheme). Por favor, ¿puedes cambiarlo? ¡Muchas gracias por la aportación!

Reply:
Lo he modificado y vuelto a subir.
Estos JWT de ejemplos los estaba generando con los test de los examples, y en ellos todos los did del configuration.js tienen el proxyAddress con 0x, habría que apuntarlo para cambiarlo.

Reply:
Los tests funcionaban entiendo que porque el 0x lo aceptaba la expresión regular, igualmente. 
En cualquier caso, y podemos concretarlo el viernes, hay dudas con qué texto tiene que admitirse como alias de la key del did, ya que algunos ponéis que tiene que haber un - otros una # y realmente no se si debemos obligar a algo o simplemente es un texto libre